R/spq_summarise.R

```diff
@@ -3,16 +3,17 @@
 #' @return A query object
 #' @export
 #' @examples
-#' result=spq_init() %>%
+#' result = spq_init() %>%
 #' spq_add("?item wdt:P361 wd:Q297853") %>%
 #' spq_add("?item wdt:P1082 ?folkm_ngd") %>%
 #' spq_add("?area wdt:P31 wd:Q1907114", .label = "?area") %>%
 #' spq_add("?area wdt:P527 ?item") %>%
 #' spq_group_by(area, areaLabel)  %>%
 #' spq_summarise(total_folkm = sum(folkm_ngd))
-spq_summarise = function(.query, ...){
+spq_summarise = function(.query, ...) {
 
   variables = purrr::map_chr(rlang::enquos(...), spq_treat_argument)
+  are_we_tallying <- (variables == "COUNT(*)")
```

Comment:
argh, should probably be an %in% as we might be creating other variables at the same time